docs/tutorial.md

```diff
@@ -260,11 +117,9 @@ You can also use the Monitoring console to create alerts and uptime checks, and
 
 ### Monitoring and logs-based metrics
 
-Cloud Logging lets you define metrics based on information in structured logs. For example, you can count the number of log entries containing a particular message or extract latency info from log records. These "logs-based metrics" can then be charted with Cloud Monitoring. For details, see ["Using logs-based metrics"](https://cloud.google.com/logging/docs/logs-based-metrics/).
+Cloud Logging defines some logs-based metrics, but you can also create your own, for details, see ["Using logs-based metrics"](https://cloud.google.com/logging/docs/logs-based-metrics/).. To see the available metrics, select **Logging> Logs-based metrics** from the navigation panel. You see a summary of the system-provided and user-defined logs-based metrics.
```

Comment:
rephrase: `Cloud Logging provides default, logs-based system metrics, but you can also create your own (see [Using logs-based metrics](https://cloud.google.com/logging/docs/logs-based-metrics/)). To see available metrics, select **Logging > Logs-based metrics** from the navigation panel. You should see both system metrics and some user-defined, logs-based metrics created in Sandbox.`